traefik/values.yaml

```diff
@@ -45,6 +45,9 @@ providers:
     enabled: true
   kubernetesIngress:
     enabled: true
+    publishedService:
```

Comment:
should we add a short description detailing what enabling this does?

Reply:
Please feel free to reopen if anything is not correct.